test/SurveyRepo-test.js

```diff
@@ -1,14 +1,38 @@
 import chai from 'chai';
+
 const expect = chai.expect;
 import SurveyRepo from '../src/SurveyRepo.js'
+import Data from '../Data/Data.js'
 
 describe('SurveyRepo', function() {
   let surveyRepo;
   beforeEach(() => {
-    surveyRepo = new SurveyRepo();
+    surveyRepo = new SurveyRepo(Data);
   });
 
   it('should be a function', function() {
     expect(SurveyRepo).to.be.a('function');
   });
-});
\ No newline at end of file
+
+  it('should be an instance of SurveyRepo', function() {
+  	expect(surveyRepo).to.be.an.instanceof(SurveyRepo);
+  });
+
+  it('should accept a file path accessing survey data', function() {
+  	expect(surveyRepo.surveys).to.be.an('array')
+  });
+
+  it('should store all questions and answers in randomized order', function() {
```

Comment:
Avoid testing for randomness as there is no such thing. I would just junk this test.